test/Twig/Tests/TemplateTest.php

```diff
@@ -234,6 +236,21 @@ public function testGetAttributeWithTemplateAsObjectForDeprecations()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation Displaying undefined block "unknown" in template "index.twig" is deprecated since version 1.29 and will throw an exception in 2.0.
+     * @expectedDeprecation Displaying undefined block "unknown" in template "index.twig" is deprecated since version 1.29 and will throw an exception in 2.0.
+     */
+    public function testRenderBlockWithUndefinedBlock()
+    {
+        // to be removed in 2.0
```

Comment:
this comment must be removed. Tests to be removed in 2.0 are all the legacy tests of 1.x, so there is not need to add this comment